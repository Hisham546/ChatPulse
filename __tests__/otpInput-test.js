import React from 'react';
import renderer from 'react-test-renderer';
import OtpInputBox from '../src/components/otpInput';

test('renders correctly', () => {
  const tree = renderer.create(<OtpInputBox />).toJSON();
  expect(tree).toMatchSnapshot();
});