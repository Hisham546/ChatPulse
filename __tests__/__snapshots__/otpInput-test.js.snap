// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flexDirection": "row",
      "height": 106.72,
      "justifyContent": "space-evenly",
      "width": 637.5,
    }
  }
>
  <TextInput
    keyboardType="numeric"
    maxLength={1}
    onChangeText={[Function]}
    style={
      {
        "borderColor": "#EFEFEF",
        "borderRadius": 5,
        "borderWidth": 2,
        "color": "#FFFFFF",
        "height": 80.03999999999999,
        "textAlign": "center",
        "width": 90,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <TextInput
    keyboardType="numeric"
    maxLength={1}
    onChangeText={[Function]}
    style={
      {
        "borderColor": "#EFEFEF",
        "borderRadius": 5,
        "borderWidth": 2,
        "color": "#FFFFFF",
        "height": 80.03999999999999,
        "textAlign": "center",
        "width": 90,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <TextInput
    keyboardType="numeric"
    maxLength={1}
    onChangeText={[Function]}
    style={
      {
        "borderColor": "#EFEFEF",
        "borderRadius": 5,
        "borderWidth": 2,
        "color": "#FFFFFF",
        "height": 80.03999999999999,
        "textAlign": "center",
        "width": 90,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <TextInput
    keyboardType="numeric"
    maxLength={1}
    onChangeText={[Function]}
    style={
      {
        "borderColor": "#EFEFEF",
        "borderRadius": 5,
        "borderWidth": 2,
        "color": "#FFFFFF",
        "height": 80.03999999999999,
        "textAlign": "center",
        "width": 90,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <TextInput
    keyboardType="numeric"
    maxLength={1}
    onChangeText={[Function]}
    style={
      {
        "borderColor": "#EFEFEF",
        "borderRadius": 5,
        "borderWidth": 2,
        "color": "#FFFFFF",
        "height": 80.03999999999999,
        "textAlign": "center",
        "width": 90,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <TextInput
    keyboardType="numeric"
    maxLength={1}
    onChangeText={[Function]}
    style={
      {
        "borderColor": "#EFEFEF",
        "borderRadius": 5,
        "borderWidth": 2,
        "color": "#FFFFFF",
        "height": 80.03999999999999,
        "textAlign": "center",
        "width": 90,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
</View>
`;
